.gitignore

```diff
@@ -0,0 +1,328 @@
+package-lock.json
+package.json
+test.js
+Login_tests/logintest.js
+Login_tests/logintest.unsuccessful.js
+node_modules/.package-lock.json
+node_modules/.bin/rimraf
```

Comment:
Use '*' to reduce amount of lines

Reply:
I used '/' to ignore an entire directory with all its contents